nf_core/pipeline-template/.github/workflows/ci.yml

```diff
@@ -1,12 +1,7 @@
 name: nf-core CI
 # This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
 on:
-  push:
-    branches:
-      - dev
```

Comment:
I disagree with the assumption that people confirmed that tests pass before opening a PR. in my experience this does not always happen.

Reply:
okay, looked a bit into the history of these checks (and the if clause, checking for the event type). these seem to be remnants from us building docker images in the CI. so I am fine with this setup.